### ISSUE TITLE
"Fake Player" crashes my Game

### ISSUE THREAD

**Issue by fallennewyork** [mchorse/blockbuster#66]
Hello,

if I use the "Fake Player" button and play my recording via the PlayButton my Game crashes. All other buttons are working without that. I think I need that, beacuse I use a Modification to make better bodys and it seems to work only with "real" players. And not with bots. So i need to use the Fake Player function.

I hope somebody can help me.

Thanks
FallenNewYork

**Comment by fallennewyork** [comment]
Hello,

Description:
If I use a -- Block and record a player (Singleplayer) and want to replay it, I can do that. But if I tick the Box by "Fake Player" the game quits my world and then with the button "cancel" to go back to Main Menu my game crashes. I use some other mods too (you can see that in the video). 

FakePlayer Crash: https://pastebin.com/B9Df0434
Bug Report Video: https://youtu.be/sqGqGT-L74I

I think this function is required for my Movie, because i use the "Mo Bends" Mod to make better bodys for the players. But that doesn´t work with bots. Maybe with "Fake Players".

Thanks

**Comment by mchorse** [comment]
Hmm... :thinking: 

That's a very weird crash... Thanks for making the video showcasing this behavior. Unfortunately, I'm still not sure what causes it, because the crash log is caused by some undefined behavior which shouldn't happen at all (it says that it crashed when updating the list of director blocks in the sidebar).

Could you also post the log file (`.minecraft\logs\latest.log` or one of those gzipped log files when that crash occurred) which was generated during the same game? If you're not sure which file to post, then do the same thing to reproduce the crash and then pastebin the `latest.log` file.

Thanks! Although I feel like the issue might be coming from ReplayMod.

**Comment by fallennewyork** [comment]
Hey,

after quitting the Game and Restarting the Error / Crash occours again.
Here is the log-file: http://crew8.cc/dll/latest.log
Login: crew10
PW: crew9

(Pastebin crashes if I paste the Text)

And without ReplayMod nothing changes.

But do you have an Idea why the Mo Bend´s Mod doesn´t effect the "Fake Player"? On a Multiplayer Server I the players were affected my the mod.

Thanks

**Comment by fallennewyork** [comment]
Hello,

is it possible to fix that Error? I need the Mod with that function 👍 

Thank you

**Comment by mchorse** [comment]
I don't know exactly. Could you test this issue with only Blockbuster and Metamorph?

**Comment by fallennewyork** [comment]
Hello,

you´re right. The Mod Works without other Mods correctly. But for my Movie some other Mods are reqired. I´ll test which mod doesn´t work with this.

Thanks.

**Comment by mchorse** [comment]
No, I was asking, specifically, whether Fake Players works on the server. Did Fake Players option works on the server fine (without other mods)?

If it works, could you please test Blockbuster and Metamorph with Mo' Bends?

**Comment by fallennewyork** [comment]
Hello,

I tested both combinations of Mods in Singleplayer, becuase I have no Server. Without other Mods all The three Mods are working fine.

~ FallenNewYork

**Comment by mchorse** [comment]
Then if they are working correctly, try finding exact mod that causes this issue. I can't help without knowing which mod in particularly messing up the game.

**Comment by fallennewyork** [comment]
Hello,

sorry for mylate answer. I think it was the "CustomNPC Mod". Beacuse i removed it from my Modpack and now it is working correctly. I removed a second one named "More Player Models Mod" maybe it was incompatible too.

But the Blockbuster Mod is great! :)
~ FallenNewYork

**Comment by mchorse** [comment]
Well, if it was one MPM or CustomNPCs, then I can't really do something about it. Both of those mods are closed source, so just don't use them.

Thanks! :slightly_smiling_face:

**Comment by fallennewyork** [comment]
Hey,

In my actual Modpack for the Movie I removed both Mods. Now most things are working.

But thanks for your time. :)

~FallenNewYork
